### ISSUE TITLE
PR checklist shouldn't imply all boxes need to be checked?

### ISSUE THREAD

**Issue by josephguillaume** [josephguillaume/hydromad#56]
# Issue: 
PR checklist shouldn't imply all boxes need to be checked?

## Issue summary:
PR checklist is used to show a progress bar, but actually not all checkboxes need to be checked.

![image](https://user-images.githubusercontent.com/1201085/85353070-bfe6c200-b54a-11ea-8835-54e03bfc6b4b.png)

## Expected Behavior
Checkboxes in PR request show what points are relevant

## Current Behavior
Default Github behaviour is to show progress bar, implying all points are relevant

## Possible Solution
Don't know

**Comment by mortisflux** [comment]
Before making the PR I deleted the "Changes to Core Features" section because that wasn't relevant to me, so overall GitHub thinks there should be 12 checks to be completed. 

The easiest way to work around this is to ask people to remove checks not relevant to them prior to making the PR. You can always double check their PR and ask why they left out particular things from the checklist.

**Comment by jnothman** [comment]
Or have a checkbox saying: This is a (remove any not relevant): xxx | yyy | zzz

**Comment by WillemVervoort** [comment]
Yes, I found that confusing as well I ended up deleting some, but then was
worried I should not delete them, so a clear statement there would help

Willem

**Comment by jnothman** [comment]
We'll find in time that some of those checkboxes are redundant with other things like automatic styling or checking of style

**Comment by josephguillaume** [comment]
Yep, let's revisit how to deal with this later - I added as an issue mainly so we come back to it eventually
